plugins/catalog-graphql/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@backstage/plugin-catalog-graphql",
+  "version": "0.1.1-alpha.17",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "generate:types": "graphql-codegen",
+    "clean": "backstage-cli clean",
+    "mock-data": "./scripts/mock-data.sh"
```

Comment:
remove?

Reply:
yep was just from the `create-backend-plugin` boilerplate.